### DISCUSSION TITLE
How to create a server for "i18next-http-backend"?

### DISCUSSION THREAD

**Discussion by mwstr** [i18next/i18next#1788] [Q&A] [answered]
I found a lot of information about the i18next backends, but nothing about how to create a server side for such backends 😢

**Comment by jamuhl** [comment, marked as answer]
https://github.com/i18next/i18next-http-middleware#add-routes

**Reply by jamuhl** [reply]
regular rest (depends on your server)

**Reply by jamuhl** [reply]
simplest: just static serve your JSON translations files using eg. https://expressjs.com/de/starter/static-files.html (depends on your server how to do so)
